trailmet/algorithms/distill/response_kd.py

```diff
@@ -1,11 +1,134 @@
+from multiprocessing import reduction
+import sys
+import numpy as np
+import pandas as pd
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import os
+from trailmet.utils import seed_everything
+from tqdm import tqdm as tqdm_notebook
 
-from trailmet.algorithms.distill import Distillation
+from trailmet.algorithms.distill.distill import Distillation
+from trailmet.algorithms.distill.losses import KDTransferLoss
 
-class Response_KD(Distillation):
+# Hinton's Knowledge Distillation
+seed_everything(43)
 
-    def __init__(self):
-        super(Response_KD, self).__init__()
+        
+class KDTransfer(Distillation):
+    """class to compress model using distillation via kd transfer"""
+    def __init__(self, teacher_model, student_model, dataloaders, **kwargs):
+        super(KDTransfer, self).__init__(**kwargs)
+        self.teacher_model = teacher_model
+        self.student_model = student_model
+        self.dataloaders = dataloaders
+        self.kwargs = kwargs
+        self.device = kwargs['DEVICE']
```

Comment:
No need of this line ~`self.device = kwargs['DEVICE']`~
yaml config file contains `cuda_id` and devices are handled by Base Distillation class